src/nft/utils/txRoute/combineItemsWithTxRoute.ts

```diff
@@ -19,16 +19,19 @@ const isTheSame = (item: GenieAsset, routeAsset: BuyItem | PriceInfo) => {
   }
 }
 
-const isPriceDiff = (oldPrice: string, newPrice: string) => {
-  return formatWeiToDecimal(oldPrice) !== formatWeiToDecimal(newPrice)
+const getPriceDiff = (oldPrice: string, newPrice: string): { hasPriceDiff: boolean; hasVisiblePriceDiff: boolean } => {
```

Comment:
nit: a potentially cleaner alternative is to remove this fn and instead have a visible threshold (ie .0001 or so) and determine behaviour if the absolute price difference is greater or less than the threshold

Reply:
well the reason this exists is because product thought it was confusing when users were getting displayed a price change but visually it was the same prices because of our rounding style. So the visible threshold is actually dependent on how we render the number.